README.md

```diff
@@ -199,5 +199,15 @@ https://graypaper.com/
   - [x] payload hash - easy
   - [x] gas prioritization - just from WP?
   - [x] Consider introducing a host-call for reading manifest data rather than always passing it in.
+### ELVES
+- [x] Don't immediately alter kappa mid-epoch as it affects off-chain judgements.
+- [x] Instead, apply the blacklist to guarantor sig verification directly.
+- [x] Include the core in the WR.
+- [x] Deposit the WR's signatures in with the judgement.
+- [x] Require at >= 1 negative judgements to be included with a positive verdict, and place signer in punish keys.
+- [x] Serialization of judgement stuff
+- [x] Should always be possible to submit more guarantee signatures of a known-bad report to place them in the punish set.
+  - [x] Only from lambda and kappa?
+- [x] Use posterior kappa for everything except judgements.
```

Comment:
I'm not sure what's going on with assurances.

In Polkadot, at least before asynchronous backing, we timed out backers pending availability at the end of a session. This was because parablock validity depended on which session we are in.

If JAM allows WPs guaranteed in the previous epoch to be assured in this one, then which set does the voting?

I'd have thought the prior set makes sense, especially if there is no block in the last slot of the previous epoch.